.github/workflows/xbox_nxdk.yml

```diff
@@ -0,0 +1,47 @@
+name: Xbox
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [ opened, synchronize ]
+
+jobs:
+  xbox:
+    # `ubuntu-latest` is currently 20.04 but we require a newer clang.
+    runs-on: ubuntu-22.04
+    env:
+      NXDK_DIR: /opt/nxdk
+    steps:
+    - name: Install APT packages
+      run: sudo apt-get update && sudo apt-get install -y clang llvm lld bison flex cmake git gettext
+
+    - name: Clone NXDK Repo
```

Comment:
nit: The preferred spelling of nxdk is "nxdk" (all lowercase, even as first word in sentence).
Initially there were some disagreements about that spelling, but we settled on the all-lowercase variant a couple of years ago (if I remember correctly, also to differentiate it from the MS XDK). All-caps `NXDK` is only used in code for defines and such.

There were other instances in this PR which use the all-caps spelling, too.

*Slightly related: While MS didn't agree on a stylization around 2001, the XboxDev community prefers to use the now-preferred MS spelling of "Xbox" (capital "X", lower-case "box") for the console name. If there might be confusion with other consoles, "original Xbox" is used ("original" in lowercase unless at start of sentence or similar).*

Reply:
Thanks, noted!